### ISSUE TITLE
Contents Help broken for xviewer, xreader, xed after upgrade to Mint 21.0 Cinammon

### ISSUE THREAD

**Issue by programmer-ceds** [linuxmint/xviewer#173]
```
Mint 21.0 Cinnamon (upgraded from 20.3)
Xviewer 3.2.10
Xed 3.2.7
Xreader 3.4.5
```
For the applications listed above selecting "Help/Contents" causes the help window to be displayed but showing "Documentation not Found"

**Comment by programmer-ceds** [comment]
I'm not sure what is going on here. I have just tried to access help in the above locations on another PC that was similarly updated from 20.3 to 21.0 and the help documentation is found. On the first system I still get the following:

/home/richard/Pictures/Screenshot from 2022-09-03 14-03-15.png

**Comment by programmer-ceds** [comment]
This seems to be a problem with my system setup rather than xviewer so closing the issue.

**Comment by programmer-ceds** [comment]
I've now realised what the problem is - somehow I have KDE Help Centre installed and in /usr/share/applications/mimeinfo.cache there are two applications listed in the line for "help":

    x-scheme-handler/help=org.kde.Help.desktop;yelp.desktop;

As the first application is the one that is called the KDE Help Centre tries, and fails, to show the help information for the applications listed in the issue title. Temporarily editing the line to read:

    x-scheme-handler/help=yelp.desktop;

fixes the problem - until mimeinfo.cache is rebuilt.

Clearly the easy solution is to uninstall khelpcenter with the Synaptic Package Manager. However, my question is should help for the x applications work with khelpcenter installed? If so the x tasks could be altered to use a call of system() to activate yelp. I have tried this and it works but it would involve making the change to all of the x applications.
